Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -816,35 +522,32 @@
             AddTabButtonClick="TabView_AddTabButtonClick"
             AllowDropTabs="True"
             CanDragTabs="True"
+			Background="Transparent"
             DragLeave="TabStrip_DragLeave"
             IsAddTabButtonVisible="True"
             SelectedIndex="{x:Bind local1:App.InteractionViewModel.TabStripSelectedIndex, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
             SelectionChanged="TabStrip_SelectionChanged"
-            Style="{StaticResource DefaultTabViewStyle}"
             TabCloseRequested="TabStrip_TabCloseRequested"
             TabDragCompleted="TabStrip_TabDragCompleted"
             TabDragStarting="TabStrip_TabDragStarting"
             TabDroppedOutside="TabStrip_TabDroppedOutside"
             TabItemsChanged="HorizontalTabView_TabItemsChanged"
             TabItemsSource="{x:Bind Items, Mode=OneWay}"
             TabStripDragOver="TabStrip_TabStripDragOver"
-            TabStripDrop="TabStrip_TabStripDrop"
-            TabWidthMode="Equal">
+            TabStripDrop="TabStrip_TabStripDrop">
             <muxc:TabView.TabItemTemplate>
                 <DataTemplate x:DataType="local:TabItem">
                     <muxc:TabViewItem
-                        Height="36"
-                        Margin="4,4"
-                        HorizontalAlignment="Stretch"
+						CornerRadius="4"
```

Comment:
Why are you using this instead of `ControlCornerRadius`? It's important to use a resource so that users can change it in custom themes.

Reply:
@yaichenbaum I was under the impression that setting that resource from App.xaml didn't work in WinUI v2 apps

Reply:
It should work, we are doing it in several other areas of the app.